code/game/objects/objs.dm

```diff
@@ -211,19 +211,14 @@
 	if(directional_offset)
 		update_directional_offset()
 
-/obj/Move()
```

Comment:
Because afterthought really wanted me to change that.
https://github.com/NebulaSS13/Nebula/pull/2674#discussion_r980650767

Reply:
I don't really know anymore tbh. I remember having a talk with you and afterthought and it didn't make much sense to intercept move since the objects updating their offsets were all anchored. 
And there too : https://github.com/NebulaSS13/Nebula/pull/2674#discussion_r980650767

Reply:
Restored that code to what it was before.